### PR TITLE
3075 literature evidence

### PR DESCRIPTION
**Pull request by ehsanb29**
Remove evidence form literature outputs.
Fix issue [3075](https://github.com/opentargets/issues/issues/3075)

### CODE REVIEW

**Comment by d0choa** [comment]
Does it make sense to consolidate the `epmc` and `literature` part of the `reference.conf`? They are conceptually the same step as far as I can tell

**Comment by ehsanb29** [comment]
> Does it make sense to consolidate the `epmc` and `literature` part of the `reference.conf`? They are conceptually the same step as far as I can tell

Yes, good idea. We can do it but needs more check on ETL. I think it would be better to raise a new issue for this.

**Comment by mbdebian** [comment]
> > Does it make sense to consolidate the `epmc` and `literature` part of the `reference.conf`? They are conceptually the same step as far as I can tell
> 
> Yes, good idea. We can do it but needs more check on ETL. I think it would be better to raise a new issue for this.

@ehsanb29 , have you raised the issue related to _epmc_ consolidation into literature?

File: configuration/2023/23_09_platform.conf
Comment (mbdebian) [inline configuration/2023/23_09_platform.conf:1]: This doesn't belong in the scope of this PR

File: configuration/2023/23_09_ppp.conf
Comment (mbdebian) [inline configuration/2023/23_09_ppp.conf:1]: This doesn't belong in the scope of this PR

File: configuration/2023/workflow_23_09.conf
Comment (mbdebian) [inline configuration/2023/workflow_23_09.conf:1]: This doesn't belong in the scope of this PR

**Comment by ehsanb29** [comment]
> > > Does it make sense to consolidate the `epmc` and `literature` part of the `reference.conf`? They are conceptually the same step as far as I can tell
> > 
> > 
> > Yes, good idea. We can do it but needs more check on ETL. I think it would be better to raise a new issue for this.
> 
> @ehsanb29 , have you raised the issue related to _epmc_ consolidation into literature?

Yes, I did. [#3099](https://github.com/opentargets/issues/issues/3099)

**Review by mbdebian — APPROVED** [review]
Thanks for the PR @ehsanb29 .
Please, keep in mind that this PR should only contain changes related to its topic, i.e. the removal of the unused _evidence_ computation, and the changes related to 23.09 release should have been PR'ed back into the repository separately.

**Pull request merged**